src/util/dom/document-height.js

```diff
@@ -0,0 +1,25 @@
+const body = document.body;
+const html = document.documentElement;
+
+/**
+ * Get the height of the whole page
+ * @link https://javascript.info/size-and-scroll-window
+ * @return {number} Height in px of the document
+ */
+export default function documentHeight() {
```

Comment:
I thought we made https://github.com/biancojs/viewport for this. Do you want to keep this project standalone?